### PR TITLE
[CARBONDATA-2414][Doc] Optimize documents for sort column bounds

### PR DESCRIPTION
**Pull request by xuchuanyin**
Optimize documents for sort column bounds

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed?
 `NO`
 - [x] Any backward compatibility impacted?
 `NO`
 - [x] Document update required?
`Updated document`
 - [x] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
`NA`
        - How it is tested? Please attach test report.
`NA`
        - Is it a performance related change? Please attach the performance test report.
`NA`
        - Any additional information to help reviewers in testing this change.
`NA`
       
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
`NA`

### CODE REVIEW

**Comment by xuchuanyin** [comment]
@chenliang613 Please review the updated document

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -488,14 +488,17 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
 
   - **SORT COLUMN BOUNDS:** Range bounds for sort columns.
 
+    Suppose the table is created with 'SORT_COLUMNS'='col1,col2,col3'. During data loading, we can specify the following option to enhance data loading performance.
```
Comment (chenliang613) [inline docs/data-management-on-carbondata.md]: Still not be easy to understand, can you please give a real example?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/4332/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5498/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4600/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/4336/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5501/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**